lnrpc/chainrpc/chain_server.go

```diff
@@ -233,9 +260,51 @@ func (r *ServerShell) CreateSubServer(configRegistry lnrpc.SubServerConfigDispat
 	}
 
 	r.ChainNotifierServer = subServer
+	r.ChainKitServer = subServer
 	return subServer, macPermissions, nil
 }
 
+// GetBlock returns a block given the corresponding block hash.
+func (s *Server) GetBlock(ctx context.Context, in *GetBlockRequest) (
```

Comment:
nit: break before `in` parameter to avoid dangling open parenthesis at line end?